var/spack/repos/builtin/packages/py-fenics-basix/package.py

```diff
@@ -29,12 +42,15 @@ class PyFenicsBasix(PythonPackage):
     depends_on("fenics-basix@0.2.0", type=("build", "run"), when="@0.2.0")
     depends_on("fenics-basix@0.1.0", type=("build", "run"), when="@0.1.0")
 
+    depends_on("cmake@3.19:", type="build")
     depends_on("python@3.7:", type=("build", "run"))
-    depends_on("py-setuptools", type="build")
-    depends_on("cmake@3.18:", type="build")
+    depends_on("py-numba", type=("run"), when="+numba")
+    depends_on("py-numpy", type=("build", "run"))
+    depends_on("py-pip", type="build")
     depends_on("py-pybind11@2.6.2:", type="build")
+    depends_on("py-setuptools", type="build")
+    depends_on("py-wheel", type="build")
```

Comment:
Not needed since it's in the `PythonPackage` base class, see https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#dependencies.